src/main/frontend/src/code-quarkus/pickers/extensions-picker.tsx

```diff
@@ -69,6 +69,11 @@ function StatusTag(props: { status?: string }) {
         className="extension-tag experimental"
         title="Early feedback is requested to mature the idea. There is no guarantee of stability nor long term presence in the platform until the solution matures."
       >EXPERIMENTAL</span>);
+    case 'deprecated':
+      return (<span
+          title="This extension has been deprecated. It is likely to be replaced or removed in a future version of Quarkus...."
```

Comment:
```suggestion
          title="This extension has been deprecated. It is likely to be replaced or removed in a future version of Quarkus."
```
Drop the ellipsis, it doesn't bring anything and could be interpreted as bad.

Reply:
Thanks, it feels good to have reviews on code.quarkus :)

Reply:
Yeah, I bring the full breadth of my expertise :].